aws_lambda_builders/workflows/java_gradle/workflow.py

```diff
@@ -36,6 +36,8 @@ def __init__(self, source_dir, artifacts_dir, scratch_dir, manifest_path, **kwar
         ]
         if self.dependencies_dir:
             self.actions.append(JavaCopyDependenciesAction(artifacts_dir, self.dependencies_dir, self.os_utils))
+            if not self.combine_dependencies:
```

Comment:
Are we able to move dependencies instead of copy and then delete?
```
if dependencies_dir:
  if self.combine_dependencies:
    copy deps
  else:
    move deps
```

Reply:
updated